types/tsconfig.json

```diff
@@ -7,6 +7,11 @@
     "noImplicitThis": true,
     "strictNullChecks": true,
     "noEmit": true,
-    "strictFunctionTypes": false
+    "strictFunctionTypes": false,
+    "baseUrl": ".",
+    "paths": {
+      "loadable-components": ["."],
```

Comment:
This is how it could be fixed!?

Reply:
What do you mean? The module resolution works out of the box as soon as this package is in node_modules, but within the repository, the compiler seems to need a little hint to be able to locate the module. I don't know if it's the best way, but it works :)

Reply:
I wrote the initial TS definitions, and fail to get dtslint working with server as a separate file.
You just fixed it, it the quite straight way, the way I could not figure out of dtslint documentation.
According to them - _everything_ should work even without patching tsconfig, and I will not be surprised if it will, but they wont 💩 :)

Reply:
Submodules are apparently more complicated :) I'm not too fond of the fact that while most type definitions are in `types/`, the `server.d.ts` suddenly has to be in the package root. But it seems to be the way to go 🤷‍♂️